chromium_src/chrome/browser/signin/account_consistency_mode_manager.cc

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/signin/account_consistency_mode_manager.h"
+
+#if defined(OS_ANDROID)
+#undef BUILDFLAG_INTERNAL_ENABLE_MIRROR
+#define BUILDFLAG_INTERNAL_ENABLE_MIRROR() (0)
+
+#undef NOTREACHED
```

Comment:
I think `base/notreached.h` should be included at the top to avoid redefining the symbol in it. 

There is a less likely danger that other headers included in account_consistency_mode_manager.cc might have NOTREACHED referenced in them, but I didn't check.

Reply:
@mkarolin - Fixed. I checked other headers for `NOTREACHED`, I couldn't find any.